### PR TITLE
Add new example script

### PR DESCRIPTION
**Pull request by bwohlberg**
Add new example script.

### CODE REVIEW

File: examples/scripts/superres_ppp_dncnn_admm.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+"""
+Create a test image by downsampling and adding Gaussian white noise.
+"""
+rate = 4  # downsampling rate
+σ = 2e-2  # noise standard deviation
+
+Afn = lambda x: downsample_image(x, rate=rate)
+s = Afn(img)
+inshape = img.shape
```
Comment (tbalke) [inline examples/scripts/superres_ppp_dncnn_admm.py:53]: May as well call then `input_shape` and `output_shape`?

Comment (bwohlberg) [reply examples/scripts/superres_ppp_dncnn_admm.py:53]: Fair point; will do.

File: examples/scripts/superres_ppp_dncnn_admm.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+fig = plot.figure(figsize=(8, 6))
+ax0 = plot.plt.subplot2grid((1, rate + 1), (0, 0), colspan=rate)
+plot.imview(img, title="Reference", fig=fig, ax=ax0)
+ax1 = plot.plt.subplot2grid((1, rate + 1), (0, rate))
+plot.imview(sn, title="Downsampled", fig=fig, ax=ax1)
+fig.show()
+
+
+"""
+Show recovered full-resolution images.
```
Comment (tbalke) [inline examples/scripts/superres_ppp_dncnn_admm.py:124]: I would show the ground truth image as well but just my humble opinion.

**Review by tbalke — APPROVED** [review]

File: examples/scripts/superres_ppp_dncnn_admm.py (hunk 2 above)
Comment (bwohlberg) [reply examples/scripts/superres_ppp_dncnn_admm.py:124]: That's the "reference" image in the previous figure.

Comment (tbalke) [reply examples/scripts/superres_ppp_dncnn_admm.py:124]: Right, although I mean in the same figure as the reconstructions.

**Pull request merged**